zingolib/src/wallet/transaction_context.rs

```diff
@@ -458,6 +458,7 @@ mod decrypt_transaction {
                             note.clone(),
                             to,
                             output_index,
+                            status.is_mempool(),
```

Comment:
why not just change add_pending_note to use status as an argument? replaci

Reply:
Well...this would allow passing a `Confirmed` status to add_pending_note, which we would then need to explicitly error-handle.

Reply:
This is marked outdated by github.